doc/reference/reference_lua/decimal.rst

```diff
@@ -70,6 +73,14 @@ post-decimal digits is necessary to get 38-digit precision.
     `Lua math library <https://www.lua.org/pil/18.html>`_,
     which returns 2.718281828459.
 
+.. _decimal-is_decimal:
+
+.. function:: is_decimal(string-or-number-or-decimal-number)
+
+    Returns "true" if specified value is decimal and "false" otherwise.
+    For example if a is 123 then ``decimal.is_decimal(a)`` returns "false". 
+    If a is ``decimal.is_decimal(decimal.new(123))`` returns "true". 
```

Comment:
@Onvember please double-check this sentence. Looks confusing.

Reply:
fixed in https://github.com/tarantool/doc/pull/1149/commits/d6c8fd8f259ba4b4b7baaa3ae9c058cea81dd441